models/models/loss_callback.py

```diff
@@ -1,8 +1,8 @@
+import keras.callbacks
+
+# TODO: check if imports below are necessary
 from dataclasses import dataclass
 from typing import Any, Callable, TypeAlias
-
-import keras
-
 from models.music_model import ProgressCallback
 
```

Comment:
Is this change even correct? Below we are using `keras.callbacks`

Reply:
reverted in 52adc5082de141ef06569d664d3a145761a6b67c (import correctness was not the main point of this PR)